packages/blockchain-api/src/config.ts

```diff
@@ -14,10 +14,33 @@ function getSecrets(deployEnv: string) {
   return envSecrets
 }
 
+export function getFirebaseAdminCreds(admin: any) {
+  // TODO: move project to celo-org-mobile
+  // until then, using serviceAccountKey for all envs
+  // tslint:disable-next-line: no-constant-condition
+  if (true /* DEPLOY_ENV === 'local' */) {
+    try {
+      const serviceAccount = require('../serviceAccountKey.json')
+      return admin.credential.cert(serviceAccount)
+    } catch {
```

Comment:
Should we catch and log the error object here?

Reply:
Yes probably, I blindly cut and pasted this from `notification-service`